### PR TITLE
Fix more tipping QA issues

### PR DESCRIPTION
**Pull request by sddioulde**
### Description

Fix more tipping QA issues.

### Dragons

n/a

### How Has This Been Tested?

local against stage

### How will this change be monitored?

n/a

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sd-tip-qa

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sd-tip-qa

**Review by dylanjeffers — APPROVED** [review]
looks great

File: packages/web/src/pages/profile-page/sagas.js [hunk 1]
```diff
@@ -148,15 +148,27 @@ function* fetchSupportersAndSupporting(userId) {
...
 
+  /**
+   * If the profile is that of the logged in user, then
+   * get all its supporting data so that when the logged in
+   * user is trying to tip an artist, we'll know whether or
+   * not that artist is already being supported by the logged in
+   * user and thus correctly calculate how much more audio to tip
+   * to become the top supporter.
+   */
+  const account = yield select(getAccountUser)
```
Comment (piazzatron) [inline packages/web/src/pages/profile-page/sagas.js:159]: Maybe worth a fn for this guy since we call it 3x

**Review by piazzatron — APPROVED** [review]

File: packages/web/src/components/tipping/tip-audio/TipAudioModal.tsx [hunk 2]
```diff
@@ -27,19 +27,37 @@ const messages = {
 const titlesMap: { [key in TippingSendStatus]?: JSX.Element | string } = {
   SEND: (
     <div className={styles.tipIconTextContainer}>
-      <IconGoldBadge width={24} height={24} />
+      <img
+        draggable={false}
+        alt=''
+        src={IconGoldBadge as string}
```
[the remark this answers is not in the document]
Comment (sddioulde) [reply packages/web/src/components/tipping/tip-audio/TipAudioModal.tsx]: copy pasta actually. we do it in UserBadges.tsx but let me see if i can remove

**Pull request merged**